credentials/forms.py

```
@@ -0,0 +1,35 @@
+from django import forms
+from credentials.models import Credential
+from datetime import datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)